src/components/Collections/Table/Table.vue

```diff
@@ -93,7 +93,6 @@
                   :class="tableRowClass"
```

Comment:
Added one more issue to the list of bugs:
- All plios shows number of plios shown in table not total number of plios

Reply:
Also, the other issue mentioned in the bug:
- Reduce size of URL in all plios home page to fit screen

will this be a separate PR?

Reply:
Add both of them in this PR